eng/scripts/CodeChecks.ps1

```diff
@@ -181,7 +187,7 @@ try {
     run 'eng\scripts\Export-API.ps1' if you changed public APIs (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#public-api-additions). `
```

Comment:
The git diff check applies for both $ServiceDirectory and $ProjectListOverrideFileName. No condition here.